azure-pipelines.yml

```diff
@@ -491,14 +509,17 @@ stages:
               inputs:
                 versionSpec: '5.8.0'
             - task: PowerShell@2
-              displayName: "Get version from artifact file"
+              displayName: "Get version from artifact file and unpack scanner"
               inputs:
                 targetType: 'inline'
                 script: |
                   $projectVersion = Get-Content "$(Pipeline.Workspace)\\scanner-packages\\version.txt"
                   Write-Host "##vso[task.setvariable variable=SONAR_PROJECT_VERSION]$projectVersion"
+
+                  Set-Location "$(Pipeline.Workspace)/scanner-packages"
+                  Get-ChildItem -Filter *net46.zip | Select-Object -First 1 | Expand-Archive -DestinationPath scanner -Force
```

Comment:
Educational: 
Why are you unzipping the scanner zip?
I noticed that the process of begin/end steps is different between SQ and SC tests:
SQ: `TestUtils.newScanner(ORCHESTRATOR, projectDir).configure(...)`
SC: `Command.create(new File(SCANNER_PATH).getAbsolutePath()).configure(...)`

Is it because it is not possible to use the orchestrator `scanner` scaffolding without doing `sonarqube` scaffolding?

Reply:
thanks for explaining!